MapboxNavigation/NavigationViewController.swift

```diff
@@ -687,6 +688,21 @@ extension NavigationViewController: NavigationServiceDelegate {
         return navigationComponents.allSatisfy { $0.navigationServiceShouldDisableBatteryMonitoring(service) }
     }
     
+    public func navigationServiceDidChangeAuthorization(_ service: NavigationService, _ manager: CLLocationManager) {
+        // CLLocationManager.accuracyAuthorization was introduced in the iOS 14 SDK in Xcode 12, so Xcode 11 doesn’t recognize it.
+        guard let accuracyAuthorizationValue = manager.value(forKey: "accuracyAuthorization") as? Int
+        else { return }
```

Comment:
FYI, it’s common to put this all on one line, or at least to put `else` on the same line as the rest of the `guard` statement.

Reply:
not sure what happened with formatting but it's technically on the same line